ckanext/canada/plugins.py

```diff
@@ -51,6 +56,9 @@ def update_config(self, config):
             "ckan.user_list_limit": 4000
         })
 
+    def wet_theme(self):
+        return 'GCWeb'
+
```

Comment:
aren't we using the same wet theme for portal and registry now? If so we could remove these methods.